R/spark_apply_bundle.R

```diff
@@ -6,7 +8,7 @@ spark_apply_bundle_file <- function(packages, base_path) {
   file.path(
     base_path,
     if (isTRUE(packages))
-      "packages.tar"
+      paste("packages_", uuid::UUIDgenerate(), ".tar", sep = "")
```

Comment:
Creating the package bundle takes a long time, so I think we should avoid creating it to only once per session.  There is a sessionId that is assigned to all connections, see https://github.com/sparklyr/sparklyr/blob/cf3f4d6e5f56856aa4f25fe887190c11b2090c8a/R/shell_connection.R#L163-L166. So I think if we reuse that we are better off to avoid slowing down subsequent calls to `spark_apply()`.

```suggestion
      paste("packages_", sc$sessionId, ".tar", sep = "")
```